$MirOS: ports/databases/postgresql/patches/patch-src_makefiles_Makefile_openbsd,v 1.1.7.1 2005/03/18 15:44:16 tg Exp $
$OpenBSD: patch-src_makefiles_Makefile_openbsd,v 1.2 2002/12/17 16:38:54 naddy Exp $
--- src/makefiles/Makefile.openbsd.orig	Wed Apr 19 18:31:45 2006
+++ src/makefiles/Makefile.openbsd	Thu Feb 22 20:28:39 2007
@@ -2,7 +2,7 @@ AROPT = cr
 
 ifdef ELF_SYSTEM
 export_dynamic = -Wl,-E
-rpath = -Wl,-R'$(rpathdir)'
+rpath = -Wl,-rpath,'$(rpathdir)'
 endif
 
 DLSUFFIX = .so
